### DISCUSSION TITLE
Modification to use existing Wi-Fi SSID

### DISCUSSION THREAD

**Discussion by keithledgerwood** [ludwig-v/wireless-carplay-dongle-reverse-engineering#63] [General]
I am interested in attempting to modify the device to use my car’s existing Wi-Fi SSID.

I would need the adapter to connect as a client instead of hosting its own AP.  Any thoughts @ludwig-v?

My goal: be able to use CarPlay AND my car’s hotspot at the same time.

**Comment by hassankhan** [comment]
Hi @keithledgerwood, I'm not sure that's possible; as I understand it the CarPlay wireless spec requires a connection between the head unit (or dongle in this case) and the device directly.

Some more info here: https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/56#discussioncomment-643579

**Reply by iDoka** [reply]
Definitely right.
Official CarPlay specs requires low enough latency for wireless link (not only codec choice, also including WiFi HW requirements)
